### PR TITLE
[ADS-5901][ASC-139] fix: add dts to image cropper

### PR DESCRIPTION
**Pull request by superpower1**
## Description

Add dts to image cropper and action panel for testing purpose
https://adslot.atlassian.net/browse/ADS-5901

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

**Review by xiaofan2406 — APPROVED** [review]

**Review by devharris7 — DISMISSED** [review]
LGTM

**Review by xiaofan2406 — DISMISSED** [review]

File: src/components/ImageCropper/index.jsx [hunk 1]
```diff
@@ -17,49 +17,56 @@ const defaultOptions = {
   toggleDragModeOnDblclick: false,
 };
 
-const ImageCropper = forwardRef(({ title, src, alt, onCancel, onCrop, width, height, aspectRatio, isSaving }, ref) => {
-  const cropperRef = React.useRef();
-  const imageRef = React.useRef();
+const ImageCropper = forwardRef(
```
Comment (xiaofan2406) [inline src/components/ImageCropper/index.jsx:20]: all these are just nesting diff right?

**Review by xiaofan2406 — APPROVED** [review]

File: src/components/ImageCropper/index.jsx (hunk 1 above)
Comment (renrizzolo) [reply src/components/ImageCropper/index.jsx:20]: Yeah, only change is dts passed to action panel

**Review by pphminions — APPROVED** [review]
LGTM 🎈

**Comment by pphminions** [comment]
merging 🚀

**Pull request merged**